preview-docs/GraphRoles.md

```diff
@@ -8,11 +8,11 @@ We've identified new requirements for access control at the graph level since th
 
 Here's what we plan to do for the next phase of work in Studio on access control, which we hope will cover your needs for graph and variant permissions. If this plan does not address your needs, please let us know so we can understand what the gaps are.
 
-1. Introduce a way to override any role a user might have in the org with a higher role for a specific graph. A user might be an **Observer** in the org, but can be promoted to a **Contributor** on a specific graph.
-2. Introduce a **Graph Admin** role that can operate with all permissions of the **Contributor** role, but can also manage user access to the graph (our current **Admin** role becomes **Org Admin**). When users create a graph, they are automatically the graph admin, and they can delegate that permission to other users by granting the graph admin role to other users on that graph.
-3. Introduce a way to mark variants as **protected**. A protected variant is effectively _read-only_ to everyone who can see the graph except the graph admins and org admins, who have permission to push schemas, see keys, change configuration, etc.
-4. Introduce a way to assign roles to graph API keys. We would make the **Consumer**, **Observer**, **Contributor**, and **Graph Admin** roles available for assignment on API key creation.
-5. Introduce a setting on graphs that flags them as "visible" or "hidden" to the members of that graph's org by default. Right now, all graphs in the org are visible to all members of the org. If you mark a graph as "hidden" to your org, only you will see it. You can grant permission for other users in your org to see your graph as well.
+1. Introduce a **Graph Admin** role that can operate with all permissions of the **Contributor** role, but can also manage user access to the graph and delete the graph. (Our current **Admin** role will be renamed to **Org Admin**. **Graph Admin**s and **Org Admin**s will have identical permissions to act on graphs, but org admins will also be able to manage org membership, billing, etc.)  The **Contributor** role will be made less powerful. It will not be able to perform management tasks on the graph like updating integrations and settings; it will have all the permissions of **Observer** plus the ability to push new versions of the graph's schema.
```

Comment:
Do we have a sense of the migration for existing Contributors? Do they become graph Admins or lose permissions?

Reply:
I think it's worth reaching out to active customers to see what they'd prefer, but yeah, probably transitioning to Graph Admin (which will be identical to the old Contributor except that they will be able to delete graphs and manage per-graph permissions) makes sense. What do you think?

Reply:
Sorry for the late reply - ideally, I think we'd pull which customers have Contributors and let them choose what to do -- keep them as less-permissioned Contributors or upgrade them to Graph Admins. In general, if we have to take the expedient route it's probably better to keep them as Contributors, take away those permissions, then tell them all they can up-level those users at their discretion. Better to take away permissions than to give them. cc @daniman for discussion on implementing the comms on this - might ask you to run with this, or at least help me get the list.